exasol_sagemaker_extension/lua/src/install_autopilot_prediction_udf.lua

```diff
@@ -94,7 +100,7 @@ function M.install_udf(schema, endpoint_name, model_conn_name, input_params, out
 		.. schema .. ".\"" .. endpoint_name .. "\""
 		.. "(" .. table.concat(input_params, ',') .. ")"
 		.. "EMITS (" .. table.concat(output_params, ',') .. ") AS\n"
-	.. "from exasol_sagemaker_extension.autopilot_prediction import AutopilotPredictionUDF\n"
+	.. "from exasol_sagemaker_extension.autopilot_prediction_udf import AutopilotPredictionUDF\n"
```

Comment:
Good to make these change now, when we below 1.0. After, 1.0 we need to think about, who updates of the extension will work and one thing to keep them easier is, that we don't break the API of the PredictionUDFs.